shared/routes/Repl/index.html

```diff
@@ -106,9 +107,23 @@
 		display: none;
 	}
 
+	h2 {
+		margin: 1em 0 0 0;
+		padding: 0 8px;
+		font-weight: 500;
```

Comment:
500 is the weight defined in the [font-face declaration](https://github.com/sveltejs/svelte.technology/blob/master/server/templates/main.css#L33-L39) — it's consistent with the rest of the site